conversation/python/sdk/conversation/tool_calling.py

```diff
@@ -0,0 +1,52 @@
+# ------------------------------------------------------------
+# Copyright 2025 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+from dapr.clients import DaprClient
+from dapr.clients.grpc._request import (ConversationInputAlpha2, ConversationMessage, ConversationMessageContent,
+                                        ConversationMessageOfUser, ConversationToolsFunction, ConversationTools)
+
+with DaprClient() as d:
```

Comment:
Lets keep the tool calling sample in the same file as the basic python qs so that there is only one app to run as Constantin did here: https://github.com/dapr/quickstarts/pull/1209

The mechanical markdown test should test the same input/output as well:
```
== APP - conversation == Tool calling input sent: What is the weather like in San Francisco in celsius?
== APP - conversation == Output message: What is the weather like in San Francisco in celsius?
== APP - conversation == No tool calls in response
```